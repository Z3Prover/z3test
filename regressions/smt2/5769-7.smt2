(declare-fun a () RoundingMode)
(declare-fun b () Float64)
(declare-fun c () Float64)
(check-sat)
(assert (= c b ((_ to_fp 11 53) a 1.6)))
(check-sat)
(get-value (a))
(get-value (b))
(get-value (c))