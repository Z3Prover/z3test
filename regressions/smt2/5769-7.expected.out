sat
sat
((a roundNearestTiesToAway))
((b (fp #b0 #b01111111111 #x999999999999a)))
((c (fp #b0 #b01111111111 #x999999999999a)))
